routes/data.js

```diff
@@ -1,8 +1,73 @@
-var express = require('express');
-var router = express.Router();
+const express = require('express');
+const router = express.Router();
+const firebase = require('firebase');
+
+if (!firebase.apps.length) {
+	firebase.initializeApp(config);
+}
+
+// Creates connection to database.
+const db = firebase.database();
+// Links to head of database.
+const ref = db.ref("live_weller");
+// Links to markets list.
+const marketsRef = ref.child("markets");
 
 router.get('/', function(req, res, next) {
 	res.render('data');
 });
 
+router.post('/', function(req, res) {
+	if (req.body.type === "general") {
```

Comment:
Long `if`, might be worth breaking off into its own function or even its own routes, e.g. `GET /general` and `GET /question`. These are just getting data as far as I can tell and I'm assuming they're POST routes because of the little `{ type, title }` object being sent in the body so breaking them off into separate GET routes should do the trick. For the `title` variable, form questions are kinda tricky to handle because of the potentially weird characters; one possible solution might be assigning IDs to questions but for now I think stripping the key on the client-side should work.